src/sentry/models/integration.py

```diff
@@ -48,6 +48,21 @@ class Meta:
         unique_together = (("organization_integration_id", "external_id"),)
 
 
+class RepositoryProjectPathConfig(DefaultFieldsModel):
+    __core__ = False
+
+    repository = FlexibleForeignKey("sentry.Repository")
+    project = FlexibleForeignKey("sentry.Project", db_constraint=False)
```

Comment:
@wedamija I wasn't sure if I should have added `db_index=False` here if the unique constraint will also create an index.

Reply:
You can, it's probably fine either way. That's really just an optimisation, and this table probably won't be large enough or high volume enough for it to be important.